raiden/network/transport/matrix/utils.py

```diff
@@ -447,56 +480,7 @@ def login_or_register(
             current_server=server_name,
         )
 
-    # password is signed server address
-    password = encode_hex(signer.sign(server_name.encode()))
```

Comment:
Can we keep at least the default (`0`) case? I think there's value in having the default userId without the suffix, and adding it only if login/registration fails